### PR TITLE
Throw error on pairSymbol larger than expected

### LINKED ISSUE #1068: pairSymbol in createpoolpair is trimmed
#### What happened:
When using `createpoolpair` pairSymbol with length over 8 chars gets trimmed: "GOLD-SILVER" -> "GOLD-SIL"

#### What you expected to happen:
Throw error when pairSymbol is larger than expected length and avoid trimming.

#### How to reproduce it (as minimally and precisely as possible):

`defi-cli createpoolpair ({"tokenA":"GOLD","tokenB":"SILVER","commission":0.01,"status":True,"ownerAddress":"<address>","customRewards":"str","pairSymbol":"GOLD-SILVER"} )
`

### PR DESCRIPTION
**Pull request by hidiego**
#### What kind of PR is this?:

/kind fix

#### Which issue(s) does this PR fixes?:

Fixes #1068

### CODE REVIEW

File: src/masternodes/mn_checks.cpp [hunk 1]
```diff
@@ -1123,6 +1123,9 @@ class CCustomTxApplyVisitor : public CCustomTxVisitor
         const auto symbolLength = height >= consensus.FortCanningHeight ? CToken::MAX_TOKEN_POOLPAIR_LENGTH : CToken::MAX_TOKEN_SYMBOL_LENGTH;
         if (pairSymbol.empty()) {
             pairSymbol = trim_ws(tokenA->symbol + "-" + tokenB->symbol).substr(0, symbolLength);
+        }
+        else if (pairSymbol.length() > symbolLength) {
```
Comment (prasannavl) [inline src/masternodes/mn_checks.cpp:1127]: Remove the `else`. Let this be a redundant check.

File: src/masternodes/mn_checks.cpp [hunk 2]
```diff
@@ -1121,11 +1121,14 @@ class CCustomTxApplyVisitor : public CCustomTxVisitor
         }
 
         const auto symbolLength = height >= consensus.FortCanningHeight ? CToken::MAX_TOKEN_POOLPAIR_LENGTH : CToken::MAX_TOKEN_SYMBOL_LENGTH;
+        if (pairSymbol.length() > symbolLength) {
```
Comment (Bushstar) [inline src/masternodes/mn_checks.cpp:1124]: For safety this should be done by fork height, after the fork if full sync is successful we can remove the fork height check.

Comment (bvbfan) [reply src/masternodes/mn_checks.cpp:1124]: Probably we don't what it at consensus level, rpc level is better.

Comment (hidiego) [reply src/masternodes/mn_checks.cpp:1123]: I'll add checks at rpc level :100:

**Review by bvbfan — APPROVED** [review]

**Comment by prasannavl** [comment]
Could you please rebase this on top of https://github.com/DeFiCh/ain/pull/1207

**Pull request merged**